docs/cron_jobs.md

```diff
@@ -0,0 +1,190 @@
+| Job order   | order in which the tasks will be executed. The tasks with the smallest order will be executed first.                                                                                             |
+| Active      | allows you to enable/disable a scheduled task.                                                                                                                                                   |
+
+We will detail the different tasks in the following chapters that will enable you to fill the other columns.
+
+### Setting up a crontab inside the Myddleware interface
+
+Just like with the job scheduler, you can use crontab to schedule tasks directly via Myddleware. To do so, click again on your username, then click on ```Crontab```.
+
+![Crontab list](images/basic_usage/crontan_list.PNG)
+
+| Column name      | Description                                                                                                                                                                                                                                                                                                                                                                       | 
+|------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Command          | your command's name                                                                                                                                                                                                                                                                                                                                                               |
+| Period           | just like the job scheduler, period is a time interval corresponding to the when your task will be executed. <br/> Here on the other hand the syntax to use is very precise. <br/> Example : */5* ** * <br/> in order of writing:  minute(0-59), hour (0-23), day of the month(1-31), month(1-12), day of the week(0-6) (Sunday to Saturday => 7 is also Sunday on some systems). |
+| Arguments        |                                                                                                                                                                                                                                                                                                                                                                                   |
```

Comment:
@CamillePMyddleware @Myddleware I need help for these sections of the table as I'm not too familiar with crontab